docs/infrastructure/cicd.md

```
@@ -0,0 +1,156 @@
+
+---
+
+## CI Pipeline — `CI.yaml`
+
+**Triggers:** Push to `main` or `dev`, and pull requests targeting `main` or `dev`.
+
+This pipeline ensures that code is always linted, tested, and buildable before it can be merged.
+
+### Jobs (run in parallel where possible)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)